.github/labeler.yml

```diff
@@ -0,0 +1,7 @@
+# PRs that only touch the docs folder
+documentation:
+- all: ["docs/**/*"]
```

Comment:
Just as improvement, editing the main README.md would not trigger it
https://github.com/fastify/fastify/pull/2924

This workflow is just life-saver 🤟